client/landing/gutenboarding/onboarding-block/designs/index.tsx

```diff
@@ -78,26 +81,29 @@ const Designs: React.FunctionComponent = () => {
 	const { setSelectedDesign, setFonts, resetFonts, setRandomizedDesigns } = useDispatch(
 		ONBOARD_STORE
 	);
-	const {
-		getSelectedDesign,
-		hasPaidDesign,
-		getRandomizedDesigns,
-		isEnrollingInFseBeta,
-	} = useSelect( ( select ) => select( ONBOARD_STORE ) );
```

Comment:
My understanding is that selector functions should usually be executed within the callback, so I've refactored this. See p7H4VZ-2Q6-p2